### PR TITLE
Feature/search with quotes and spaces

### LINKED ISSUE #66: Other advanced search functionality
Need to double check with History:
* Ignore symbols in original texts and translations when searching, eg `{ } [ ] ( ) / \ ~ , ; : . < >` 
* How about quoting to multi-word search? E.g. search for "On Farming" without seeing all the entries that contain "on" or "farming"

### PR DESCRIPTION
**Pull request by tim-band**
Searching for multiple words searches for items containing all of those words in any order. You can search for exact strings by using double quotes.

### CODE REVIEW

**Comment by tim-band** [comment]
Just added removal of punctuation and other common symbols.

**Review by raquelalegre — APPROVED** [review]
I've only tested locally and briefly when it was deployed to pre-prod, but functionality was good.

File: README.md [hunk 1]
```diff
@@ -312,6 +312,7 @@ DJANGO_SETTINGS_MODULE=config.settings.development
 DJANGO_SECRET_KEY=<secret key>
 DJANGO_ADMIN_URL=admin/
 DJANGO_ALLOWED_HOSTS=<domain name>
+URL_PREFIX=history/frrant-preprod/
```
Comment (raquelalegre) [inline README.md:315]: Why is this in this PR? Should it be reverted?

File: README.md [hunk 2]
```diff
@@ -348,6 +349,10 @@ for the username/password to use for postgres database.
 
 If your static files have changed in a deployment (css, js, fonts etc) then you need to run the 'collectstatic' command described earlier.
 
+This development version will appear at the path
+`/history/frrant-preprod`. You can deploy it on a
+development machine as well, and it will appear
+at the path `https://localhost/history/frrant-preprod/`
```
Comment (raquelalegre) [inline README.md:355]: Does this need to be reverted?

**Pull request merged**